kinto_http/session.py

```diff
@@ -69,12 +70,15 @@ def request(self, method, endpoint, data=None, permissions=None,
         while retry >= 0:
             resp = requests.request(method, actual_url, **kwargs)
             retry = retry - 1
+            headers = resp.headers
+            if hasattr(headers, "Alert"):
```

Comment:
Yes, you're right. It should be `if "Alert" in headers` since "Alert" is not an attribute of headers. Thanks! :)